packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -969,11 +969,6 @@ test('interest on multiple vaults', async t => {
     ),
     AmountMath.make(runBrand, 3200n + bobAddedDebt),
   );
-  t.deepEqual(bobUpdate.value.interestRate, rates.interestRate);
```

Comment:
Are these getting tested in the vaultManager's notifier?

Reply:
No, nothing in agoric-sdk is reading these removed values. I'll start some new tests.

Reply:
I think Dean is asking a different question. These values are now superfluous in `vault.js` because all the relevant work is done in `vaultManager.js`. I checked, and the values are visible to the UI from the versions that would remain in `vaultManager.js`.  The question is whether there are tests corresponding to the one removed here that includes verifying the new state of the values in `vaultManager`'s updates.

Reply:
> The question is whether there are tests corresponding to the one removed here that includes verifying the new state of the values in vaultManager's updates.

Yep, that was my understanding. The answer to that question is "no" and that's why I'm going to "start some new tests".

Reply:
> The question is whether there are tests corresponding to the one removed here that includes verifying the new state of the values in vaultManager's updates.

I said "no" but that was wrong. Here the assetNotifier from vaultManager is read: https://github.com/Agoric/agoric-sdk/blame/540978c997e30c25905ea1fb837ae701db6f3536/packages/run-protocol/test/swingsetTests/governance/vat-alice.js#L47-L63